dimagi/utils/pdf/pricing.py

```diff
@@ -395,4 +406,4 @@ def get_pricing_pdf(groups, is_monthly=False):
     ))
 
     doc.build(story)
-    return data
+    return data
```

Comment:
please add a blank new line at the end of file? why was this removed?

Reply:
Editor mistake.